### PR TITLE
quick fix to allow use of the content url domain as the domain for th…

### PR DESCRIPTION
**Pull request by dallen42**
…e v1/metadata call to allow calling ssai staging api for metadata

### CODE REVIEW

File: js/ssai_pulse.js [hunk 1]
```diff
@@ -776,6 +777,7 @@ OO.Ads.manager(function(_, $)
       var mainUrl = urlParts[0];
       var mainUrlParts = mainUrl.split("/");
       if (mainUrlParts !== null) {
+      	this.domainName = mainUrlParts[2];
```
Comment (aeng7) [inline js/ssai_pulse.js]: Can we update the jsdoc to include that this function also parses for the domain name?

**Comment by aeng7** [comment]
LGTM other than a minor comment about jsdocs

File: js/ssai_pulse.js (hunk 1 above)
Comment (dallen42) [reply js/ssai_pulse.js]: @aeng7 added

**Comment by aeng7** [comment]
LGTM

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**